api/apps/api/src/modules/scenarios/marxan-run/run.service.ts

```diff
@@ -44,14 +45,18 @@ export class RunService {
     private readonly apiEvents: ApiEventsService,
     @InjectRepository(Scenario)
     private readonly scenarios: Repository<Scenario>,
+    private readonly assets: AssetsService,
   ) {
     queueEvents.on(`completed`, ({ jobId }) => this.handleFinished(jobId));
     queueEvents.on(`failed`, ({ jobId }) => this.handleFailed(jobId));
   }
 
   async run(scenarioId: string): Promise<void> {
+    const assets = await this.assets.forScenario(scenarioId);
+    assertDefined(assets);
```

Comment:
As above, we shall keep it in top service which already does that (btw that assert wouldn't ever trigger, as AsserService does the check as well)

Reply:
it's a leftover, thanks :)